modules/KalturaSupport/resources/mw.kAnalony.js

```diff
@@ -144,13 +144,14 @@
                 }
 
 				if ( !this.isInSequence() && (_this.firstPlay || _this.embedPlayer.currentState !== "play") ){
-					if ( _this.firstPlay ){
+					if ( _this.firstPlay && !_this.onPlayStatus ) {
+						_this.onPlayStatus = true;
```

Comment:
need to reset the flag state on change media otherwise it will never enter here after first time

Reply:
@OrenMe Added it to the 'onChangeMedia' function.